### ISSUE TITLE
8.2 CE image, SONAR_WEB_CONTEXT doesn't work in docker-compose

### ISSUE THREAD

**Issue by andav51** [SonarSource/docker-sonarqube#387]
I configured SonarQube in my docker-compose file to have Sonar run under the `/sonarqube` context. To achieve this, I first tried using the `SONAR_WEB_CONTEXT` environment variable but after launching my services, Sonar was still accessible at the root context and not `/sonarqube`. I then tried defining the context in `sonar.properties` and mounted `sonar.properties` in `/opt/sonarqube/conf`. That worked. So it seems to me that `SONAR_WEB_CONTEXT` is ignored. My service definition from my compose file is included below.

```
  sonarqube:
    container_name: sonarqube
    image: sonarqube:8.2-community
    expose:
      - "9000"
    depends_on:
      - sonarqube-db
    volumes:
      - sonarqube-data:/opt/sonarqube/data
      - sonarqube-extensions:/opt/sonarqube/extensions
      - sonarqube-logs:/opt/sonarqube/logs
      - ./sonar.properties:/opt/sonarqube/conf/sonar.properties
    environment:
      #- SONAR_WEB_CONTEXT=/sonarqube # Not working.
      - SONAR_WEB_PORT=9000
      - SONAR_JDBC_URL=jdbc:postgresql://sonarqube-db:5432/sonarqubedb
      - SONAR_JDBC_USERNAME=sonarqubedbuser
      - SONAR_JDBC_PASSWORD=sonarqubedbpassword
    networks:
      - network
```

**Comment by michal-duda-sonarsource** [comment]
Indeed, I confirm that `SONAR_WEB_CONTEXT` and also `SONAR_LOG_LEVEL` environment variables do not work in current version of SonarQube :(.

As a temporary replacement for `SONAR_WEB_CONTEXT` you can pass `-Dsonar.web.context=...` to the entrypoint (for example like this: `docker run -p 9000:9000 sonarqube:8.2-community -Dsonar.web.context=/my-context`) or use `sonar.properties` with `sonar.web.context=...` setting.

Instead of `SONAR_LOG_LEVEL` you can use the following environment variables to set levels of 4 more specific loggers:
- `SONAR_LOG_LEVEL_APP`
- `SONAR_LOG_LEVEL_WEB`
- `SONAR_LOG_LEVEL_CE`
- `SONAR_LOG_LEVEL_ES`

**Comment by michal-duda-sonarsource** [comment]
Fixed in SonarQube 8.3.
